reactivity-foundations.Rmd

```diff
@@ -303,8 +303,7 @@ Outputs are intimately connected to the browser, so there's no way to experiment
 
 ## Timed invalidation {#timed-invalidation-adv}
 
-There's one last building blocking that to discuss.
-It's a little different to the others in this chapter because it's not a type of object, but is instead a special behaviour: timed invalidation.
+There's one last building block to discuss. It's a little different to the others in this chapter because it's not a type of object, but is instead a special behaviour: timed invalidation.
```

Comment:
```suggestion
There's one last building block to discuss.
It's a little different to the others in this chapter because it's not a type of object, but is instead a special behaviour: timed invalidation.
```